src/main/java/org/jabref/gui/preferences/FileTab.fxml

```diff
@@ -43,7 +43,7 @@
         <TextField fx:id="mainFileDir" HBox.hgrow="ALWAYS"/>
         <Button onAction="#mainFileDirBrowse" text="%Browse"/>
     </HBox>
-    <CheckBox fx:id="useBibLocationAsPrimary" text="%Use the BIB file location as primary file directory">
+    <CheckBox fx:id="useBibLocationAsPrimary" text="%Search for files relative to BIB file location">
```

Comment:
```suggestion
    <CheckBox fx:id="useBibLocationAsPrimary" text="%Search and store files relative to BIB file location">
```

Reply:
and maybe replace "bib file" with "library file" to be consistent with the more recent usage

Reply:
done